content/en/blog/_posts/2021-08-30-volume-populators-alpha.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Volume Populators Redesign
+date: 2021-08-30
```

Comment:
Please update the front matter; I suggest:
```suggestion
layout: blog
title: "Kubernetes 1.22: A New Design for Volume Populators"
date: 2021-08-30
slug: volume-populators-redesigned
```
(the title I'm flexible on. I do want to see a `slug` set and the `layout` explicitly declared)